http/server.go

```diff
@@ -54,14 +54,15 @@ func (s *Server) Run(done <-chan struct{}) error {
 		}
 	}()
 
-	fmt.Printf("Server started\n")
+	// TODO replace with logging
+	// fmt.Printf("Server started\n")
 
 	select {
 	case err := <-errors:
 		return err
 	case <-done:
-		fmt.Printf("Server stopped\n")
-
+		// TODO replace with logging
```

Comment:
if logging is not a part of this PR, and the `fmt` statements are useful, let's keep them in and add simply the comments above